PWGHF/TableProducer/candidateSelectorOmegac0ToOmegaPi.cxx

```diff
@@ -366,6 +397,79 @@ struct HfCandidateSelectorToOmegaPi {
         registry.fill(HIST("hSelPtPiFromCharm"), 1);
       }
 
+      // KFParticle Preselections(kfsel)
+      if (applyKFpreselections) {
+
+        bool inputKF = false;
+        if (resultSelections) {
+          inputKF = true;
+          registry.fill(HIST("hSelKFstatus"), 0);
+        }
+
+        //  Competing Ξ rejection(KF)
```

Comment:
I would try to make it clear in the code with some comments that your intention is to reject cases in which the candidate has a an inv. mass compatibler to Xi (bachelor pion) instead of Omega (bachelor kaon)